MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,781 @@
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.Background = Brush.Azure; 
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+#### Setting order for editor
+
+You can change the row order of an item using `RowOrder` property of `DataFormViewItem`.
+
+You can change the order of item's within a row using `ItemsOrderInRow` property, which is applicable only when `ColumnCount` is set to more than one. You can place an item within a row from 0 to `ColumnCount` - 1 position.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.ColumnCount = 3;
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 1;
+                e.DataFormItem.ItemsOrderInRow = 2;  
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Setting Label icon
+
+You can set label icon instead of using label text. 
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "PhoneNumer")
+            {
+                e.DataFormItem.LeadingLabelIcon = new FontImageSource() 
+                { 
+                    Glyph = "K", 
+                    Color = Colors.Black, 
+                    FontFamily = "DataFormIcons", 
+                    Size=18 
+                };
+            }
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+## Changing Text style 
+
+You can change the text style for label text, editor text , valid message text and error message text 
+<table>
+<tr>
+<th>Text</th>
+<th>Property</th>
+</tr>
+<tr>
+<td>Label Text</td>
+<td>LabelTextStyle</td>
+</tr>
+<tr>
+<td>Editor Text</td>
+<td>EditorTextStyle</td>
+</tr>
+<tr>
+<td>Error Message Text</td>
+<td>ErrorLabelTextStyle</td>
+</tr>
+<tr>
+<td>Valid Message Text</td>
+<td>ValidMessageLabelTextStyle</td>
+</tr>
+</table>
+
+
+by using DataFormTextStyle class properties, you can change the text style .
+
+DataFormTextStyle class properties:-
+    1.FontSize
+    2.FontFamily
+    3.FontAttributes
+    4.TextColor
+
+## Manually create DataForm Items
+
+Data form autogenerate the editors based on the data type, you can explicitly add the data editors by adding `SfDataForm.Items` manually and you need to change AutoGenerateItems property to false.
+
+Note:-Use the FieldName property to bind the editor to the data object property.
```

Comment:
Also mention in note the `GenerateDataFormItem` event will not trigger when AutoGenarte false

Reply:
added